draft-release-notes.md

```diff
@@ -0,0 +1,223 @@
+# [DRAFT]: Release Notes
+
+What's coming up? Look at `src/oas.md` on the relevant development branch for full details.
+
+## 3.2 Updates
+
+All the changes in the OpenAPI 3.2 release.
+
+### Headline features
+
+A summary of the biggest changes.
+
+#### Nested, multipurpose tags
+
+- New `summary` field to allow short descriptions, used when displaying lists of tags. If you use the `x-displayName` extension, you can now use `summary` instead.
```

Comment:
should we identify which tool(s) this exension is for?

Reply:
I think that an exhaustive list would be very difficult - I certainly couldn't get this correct without a lot of research. And the tools don't matter too much - I tried to word it so that you know what to look for in your OpenAPI description and how to update that.